### ISSUE TITLE
Two-pool mixing model?

### ISSUE THREAD

**Issue by bpbond** [PNNL-TES/rcq10#5]
We could take the priors (i.e. Q10 data by RC_annual) and take a Bayesian approach to estimate 'pure' Ra and Rh values. Talk to Alexey about this?

**Comment by jinshijian** [comment]
Yeah, that will be great. A question on this, if we get the Rh and Ra Q10s from publication, will the Rh and Ra Q10 values from publication a support for the 'pure' Rh and Ra by Bayesian?

**Comment by bpbond** [comment]
I think they will help, yes. There will be many fewer data points, but they'll be a 'pure' estimate of Rh Q10.

**Comment by jinshijian** [comment]
Sounds good, you mentioned the 'publication list with Rh and Ra separated' yesterday, can you share that with me? I want to go through those papers and collect the Q10 of Rh and Ra if reported (if not reported, I will simulate an exponential equation and calculate the Q10 if enough data points available, n>7?). Do you think it worth to do it now? I don't think it will take too much time (depend on how many papers we have).

**Comment by bpbond** [comment]
Here's the list I've been keeping (numbers are SRDB study numbers):
```
	Separate R10, Q10 for Ra and Rh 
		#6540, #5526 #5524 #5522, #5317, #5824, #6090, 5866, 
		5927, 6029, 5876, 4340, 6334, 7517, 7587, 7150, 6548, 6822, 
		7491, 8896, 10087, 10104, 10342, 10075, 10351, 10362, 7527, 
		7868, 7477, 7721, 7678, 7689
```
We can probably find more via literature searches. 

Yes, I do think it's worth it, and wouldn't take a lot of time. Maybe use a structure similar to SRDB (with `Q10_0_10`, `Q10_5_15`, `R10`, etc) but for RH only?

**Comment by jinshijian** [comment]
Thanks, will do, only for Rh is because Rh is usually directly measured, and Ra are calculated by Rs-Rh, right? But I may do both, we can decide whether to use the Ra data or not later?

**Comment by bpbond** [comment]
Right, and I agree, that makes sense.
